keps/sig-storage/3476-volume-group-snapshot/README.md

```diff
@@ -726,17 +787,34 @@ _This section must be completed when targeting beta graduation to a release._
   Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
   checking if there are objects with field X set) may be a last resort. Avoid
   logs or events for this purpose.
-
-* **What are the SLIs (Service Level Indicators) an operator can use to determine
-the health of the service?**
-  - [ ] Metrics
-    - Metric name:
-    - [Optional] Aggregation method:
-    - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
-
-* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  The following metric is added to snapshot-controller for volume group snapshot
+  operations.
+  - snapshot_controller_operation_total_seconds
+  This metric has operation_status that is either "Success" or "Failure".
+  "operation_name" can be the following:
+    - CreateGroupSnapshot (CreationTime field set to be non-nil for
+      dynamic provisioning)
+    - CreateGroupSnapshotAndReady (ReadyToUse set to true for both dynamic
+      provisioning and pre-provisioning)
+    - DeleteGroupSnapshot
+
+  The following metric is available for volume group snapshot sidecar.
+  - csi_sidecar_operations_seconds
+  This has a grpc_status_code that shows the operation status. A grpc_status_code
+  of "OK" indicates the operation is successful, otherwise, it is a failure.
+
+* **How can someone using this feature know that it is working for their instance?
+ * Events
+  * Event Reason: A GroupSnapshotCreated Normal Event with a message:
+    GroupSnapshot <name> was successfully created by the CSI driver.
+ * API .status
+  * Condition name:
+  * Other field: VolumeGroupSnapshot Status has CreationTime != nil and
+    ReadyToUse = true.
+ * Other (treat as last resort)
+  * Details:
```

Comment:
nit: indent the bullet points by more spaces (check rendered .md file on github)

Reply:
Done